lib/rails_multitenant/global_context_registry.rb

```diff
@@ -225,5 +225,9 @@ def globals=(value)
       value
     end
 
+    def integer_class
+      @integer_class ||= 0.class == Integer ? Integer : Fixnum
+    end
```

Comment:
Super minor optimization, but could this be a constant that's evaluated on file load instead of the first call to `duplicate_registry`?